packages/lit-dev-content/site/tutorials/content/async-directive/03.md

```diff
@@ -0,0 +1,39 @@
+Before we make our directive update asynchronously, first let's get a 3rd-party
+library for formatting string durations.
+
+The `timeago` npm package accepts a date, and returns a string describing the
+time that has elapsed since that date in human-readable form, like "just now" or
+"5 minutes ago" or "3 years ago", etc..
+
+Add the package to `package.json`:
+
+```json
+    "timeago.js": "^4.0.2"
+```
+
+Then import the `format` helper into your directive module:
+
+```ts
+import {format} from 'timeago.js';
+```
+
+Last, update the render method to use `format`:
+
+```ts
+    return format(time);
+```
+
+You should now see the string updated to say "just now". But it doesn't yet
```

Comment:
Maybe suggest the user change the date to `new Date('2020')` to see how it works?

Reply:
I tried putting that in, but it messes up the flow. I really like pausing to differentiate a sync function/directive from an async directive, and this interrupts that progression of points.

Similar to the point about the date picker, once you change the date to something more than a minute ago, you lose the point about seeing anything happen async.